### DISCUSSION TITLE
It not compatible with Laravel 11

### DISCUSSION THREAD

**Discussion by mahdirezaei-dev** [shopperlabs/framework#250] [General]
Unfortunately, it is not compatible with Laravel 11.

Will it be fixed?

**Comment by bdsumon4u** [comment]
It depends on:
https://github.com/shopperlabs/shopper/blob/694a5c370f91f9c61a1e3f4df736533a1ef8577b/composer.json#L49
which works with Laravel 9.x at most.
So, technically this package should not work with Laravel 9+.
`stevebauman/location` version must be updated.
